system_meta/management/tests/add_system_test.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for the add_system command"""
+
+from io import StringIO
+
+import pytest
+from django.core.management import call_command
+from factory import fuzzy
+
+from system_meta.models import IntegratedSystem
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.mark.parametrize("withDescription", [True, False])
+@pytest.mark.parametrize("withDeactivate", [True, False])
+def test_add_system(withDescription, withDeactivate):
```

Comment:
These variables aren't following snake case, is the linter not setup to check these?

Reply:
`N803` was in the ignore list, which would have caught this, so I pulled it. (FWIW, I just grabbed this out of Open and it appears [that's set that way there too](https://github.com/mitodl/mit-open/blob/9c27cbee3135703302f1138bb444773686bebac5/pyproject.toml#L177).)